_vector-search/optimizing-storage/binary-quantization.md

```diff
@@ -117,6 +117,53 @@ PUT my-vector-index
 ```
 {% include copy-curl.html %}
 
+## Enhancing search quality with ADC and RR
+
+If the recall is lacking, you can specify asymmetric distance computation (ADC) or random rotation (RR) in the index mapping to apply enhanced search capabilities. These techniques are available in OpenSearch 3.2 and later versions.
+
+ADC maintains a full-precision query vector while rescaling it to have meaningful distance computations against binary-quantized document vectors. This asymmetric approach preserves more information about the query vector, boosting search quality without significant memory penalty. ADC is supported for 1-bit quantization only.
+
+Random rotation addresses the issue where binary quantization gives equal weight to each vector dimension during the quantization process. By rotating the distribution, RR can "smooth" variance (information) from high-variance dimensions into low-variance dimensions, preserving more information during the 32x compression process. RR is supported for 1-bit, 2-bit, and 4-bit quantization.
```

Comment:
```suggestion
Random rotation addresses the issue where binary quantization gives equal weight to each vector dimension during the quantization process. By rotating the distribution, RR can balance variance (information) from high-variance dimensions into low-variance dimensions, preserving more information during the 32x compression process. RR is supported for 1-bit, 2-bit, and 4-bit quantization.
```

Reply:
Maybe "balance", "redistribute", or "reallocate" instead of "smooth"?

Reply:
Good point, I think "redistribute" is most accurate